packages/core/src/browser/icon-theme-service.ts

```diff
@@ -65,8 +65,8 @@ export class NoneIconTheme implements IconTheme, LabelProviderContribution {
         return Number.MAX_SAFE_INTEGER;
     }
 
-    getIcon(): string {
-        return '';
+    getIcon(): string | undefined {
+        return undefined;
```

Comment:
Could you elaborate please whether it is necessary?

Reply:
It looks like this is unnecessary change...

At some point, when I was debugging the label provider I found that when `LabelProvider` searches for an icon provider: https://github.com/eclipse-theia/theia/blob/master/packages/core/src/browser/label-provider.ts#L217 - the first contribution found by 
```
const contributions = this.findContribution(element);
```
is `NoneIconTheme`.
As `NoneIconTheme.getIcon()` always returns empty string (not `undefined`) this value is always returned as an icon string and no other provider is acquired for the icon value. So, I've got the tree without icons at all.
 This was the reason I changed the return value of `NoneIconTheme.getIcon()` to be `undefined`.
Not sure why did it happen to me, but when I test it now (without this change) - all works fine and `NoneIconTheme` doesn't appear in the results of `findContribution(element)`.

I'm rolling back this change.